src/engine/config/src/lib.rs

```diff
@@ -137,6 +138,7 @@ impl Default for LogConfig {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde", serde(default))]
 pub struct EngineConfig {
+    pub translation_layer: Option<Box<Path>>,
```

Comment:
Why not use `Option<PathBuf>`?

Reply:
I'm new to Rust. I did what rustc told me to. I'll update